projects/experimental/components/compass/compass.component.ts

```diff
@@ -0,0 +1,21 @@
+import {ChangeDetectionStrategy, Component, Inject, Input} from '@angular/core';
+import {MODE_PROVIDER, TUI_MODE, TuiBrightness} from '@taiga-ui/core';
+import {Observable} from 'rxjs';
+
+@Component({
+    selector: 'tui-compass',
+    templateUrl: './compass.component.html',
+    styleUrls: ['./compass.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '($.data-mode.attr)': 'mode$',
```

Comment:
Are we sure it's worth it to drag mode in here just to disable shadow? Even though it says dark mode has no shadow in figma, it actually has it and nobody noticed :D It's fairly easy to remove shadow, let's keep this component as simple as possible and not add mode at this time.

Reply:
fixed